src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -2,0 +1,22 @@
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../services/getUserById';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = getUserById(todo.userId);
```

Comment:
[CODE STYLE] - Prepare data in one place (App component) and pass it to child components. Avoid using `import` keyword in non-root components to directly fetch data. Instead, lift the state up to the parent component and pass the necessary data down as props.